compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -2025,6 +2013,20 @@ private BType visitCollectionType(BLangInputClause bLangInputClause, BType colle
         return symTable.semanticError;
     }
 
+    private BType getTupleMemberType(BType collectionType) {
```

Comment:
I think it will be clearer if we can change this function signature to accept `BTupleType` and cast when calling this function.

Reply:
Updated 3d99239eb73c2db4d5ff6fb2e905106a32b25e1d